### PR TITLE
feat(schedule): optimize log print

### PR DESCRIPTION
**Pull request by devhaozi**
## 📑 Description

Closes https://github.com/goravel/goravel/issues/443

## ✅ Checks

- [x] Added test cases for my code

### CODE REVIEW

File: schedule/application.go [hunk 1]
```diff
@@ -10,21 +10,22 @@ import (
...
 	"github.com/goravel/framework/support/carbon"
-	"github.com/goravel/framework/support/color"
 )
 
 type Application struct {
 	artisan console.Artisan
 	cache   cache.Cache
 	cron    *cron.Cron
 	log     log.Log
+	logInfo bool
```
Comment (hwbrzzl) [inline schedule/application.go:20]:
```suggestion
	debug bool
```

**Review by hwbrzzl — COMMENTED** [review]
LGTM, just optimize the variable name.

File: schedule/application_test.go [hunk 2]
```diff
@@ -37,7 +41,7 @@ func (s *ApplicationTestSuite) TestCallAndCommand() {
 	delayIfStillRunningCall := 0
 	skipIfStillRunningCall := 0
 
-	app := NewApplication(mockArtisan, nil, mockLog)
+	app := NewApplication(mockArtisan, nil, mockLog, mockConfig.GetBool("app.debug"))
```
Comment (kkumar-gcc) [inline schedule/application_test.go:44]: S: Do we really need to use `mockConfig`? We can pass `false` directly since we are not testing the config.

**Review by kkumar-gcc — APPROVED** [review]
LGTM

**Pull request merged**